src/main/kotlin/org/jetbrains/bio/big/Bed.kt

```diff
@@ -59,9 +59,11 @@ data class BedEntry(
      * @param delimiter Custom delimiter for malformed data
      * @param omitEmptyStrings Treat several consecutive separators as one
      */
-    fun unpack(fieldsNumber: Byte = 12, extraFieldsNumber: Int? = null,
-               delimiter: Char = '\t',
-               omitEmptyStrings: Boolean = false): ExtendedBedEntry {
+    fun unpack(
+            fieldsNumber: Byte = 12, extraFieldsNumber: Int? = null,
+            delimiter: Char = '\t',
```

Comment:
according to Kotlin code style each arg should be on a separate line

Reply:
Curiously, this applies to function definition, but not to the invocation; that's what got me confused.
("Method call" paragraph says "Group multiple closely related arguments on the same line.") Will fix.